addons/pos_mrp/models/pos_order.py

```diff
@@ -27,6 +27,6 @@ def _get_pos_anglo_saxon_price_unit(self, product, partner_id, quantity):
         for comp in components:
             price_unit = super()._get_pos_anglo_saxon_price_unit(comp[0].product_id, partner_id, comp[1]['qty'])
             price_unit = comp[0].product_id.uom_id._compute_price(price_unit, comp[0].product_uom_id)
-            qty_per_kit = comp[1]['qty'] / bom.product_qty / quantity
+            qty_per_kit = comp[1]['qty'] / bom.product_qty / (quantity or 1)
```

Comment:
can `bom.product_qty` also be `0` ?

Reply:
@vlst-odoo No, it's not possible to set the quantity zero. You will get a Validation Error.